src/ZPublisher/cookie.py

```diff
@@ -242,6 +242,10 @@ def domain_converter(value):
         return value
     from encodings.idna import ToASCII
     from encodings.idna import nameprep
```

Comment:
@d-maurer Is it necessary to do these imports here instead of at the top of the module?

Reply:
I don't see a good reason for that, just move them to the top.